core-bundle/contao/languages/en/default.xlf

```diff
@@ -2156,6 +2168,21 @@
       <trans-unit id="MSC.lightMode">
         <source>Light mode</source>
       </trans-unit>
+      <trans-unit id="MSC.altchaFooter">
+        <source>Protected by <a href="https://altcha.org/" target="_blank">ALTCHA</a></source>
+      </trans-unit>
+      <trans-unit id="MSC.altchaLabel">
+        <source>I'm not a robot</source>
+      </trans-unit>
+      <trans-unit id="MSC.altchaVerified">
+        <source>I'm human</source>
+      </trans-unit>
+      <trans-unit id="MSC.altchaVerifying">
+        <source>Verifying...</source>
+      </trans-unit>
+      <trans-unit id="MSC.altchaWaitAlert">
+        <source>Verifying... please wait.</source>
+      </trans-unit>
```

Comment:
```suggestion
      <trans-unit id="MSC.altchaVerifying">
        <source>Verifying …</source>
      </trans-unit>
      <trans-unit id="MSC.altchaWaitAlert">
        <source>Verifying … please wait.</source>
      </trans-unit>
```

In other translations we use proper ellipsis with a separating space, thus for consistency I think we should use that here too.